pkg/runner/runner.go

```diff
@@ -538,6 +544,52 @@ func getCellValueByLimit(cell interface{}) string {
 	return cellValue
 }
 
+func outputCveExplained(cves []types.CVEData) {
+	for _,cve := range cves {
+			// using gologger.Silent() to allow users to combine -explain and -silent flags
+			gologger.Silent().Msgf("CVE ID: %s", cve.CveID)
+			gologger.Silent().Msgf("Description: %s", cve.CveDescription)
+			gologger.Silent().Msgf("CVSS Score: %.1f", cve.CvssScore)
+			gologger.Silent().Msgf("Severity: %s", cve.Severity)
+			if len(cve.Weaknesses) != 0 {
+					for _,cwe := range cve.Weaknesses {
+							if strings.Compare(cwe.CWEID, "NVD-CWE-noinfo") != 0 {
+								gologger.Silent().Msgf("CWE Info: %s (%s)", cwe.CWEID, cwe.CWEName)
+							}
+					}
+			}
+			gologger.Silent().Msgf("Age: %d", cve.AgeInDays)
+			gologger.Silent().Msgf("Vulnerability Status: %s", cve.VulnStatus)
+			gologger.Silent().Msgf("Exploited Remotely: %t", cve.IsRemote)
+			gologger.Silent().Msgf("POC Available: %t", cve.IsPoc)
+			if cve.IsPoc {
+					gologger.Silent().Msgf("POC(s):")
+					for _,poc := range cve.Poc {
+							gologger.Silent().Msgf("\t%s - %s", poc.Source, poc.URL)
+					}
+			}
+			if len(cve.Patch) != 0 {
+					gologger.Silent().Msgf("Available Patch(es):")
+					for _,patch := range cve.Patch {
+							gologger.Silent().Msgf("\t- %s", patch)
+					}
+			}
+			if cve.IsTemplate {
+					gologger.Silent().Msgf("Nuclei Template:")
+					gologger.Silent().Msgf("\tPath: %s", cve.NucleiTemplates.TemplatePath)
+					gologger.Silent().Msgf("\tURL: %s", cve.NucleiTemplates.TemplateURL)
+			}
+			if len(cve.Reference) != 0 {
+					gologger.Silent().Msgf("Reference(s):")
+					for _,ref := range cve.Reference {
+							gologger.Silent().Msgf("\t- %s", ref)
+					}
+			}
+			gologger.Silent().Msgf("\n")
+	}
+	gologger.Silent().Msgf("For all CVE data, output to JSON using -j/-json")
```

Comment:
This might confuse users, we can add validation to warn people that `-explain` and `-json` can't be used together